docs/release_process.md

```diff
@@ -12,52 +11,120 @@ Everyone is welcome to test the stabilization branch and propose fixes.
+must be updated to refer the next release.
+    - For example, when the `stabilization-v0.1.65` is created, all open Pull Requests with the
+    Milestone `0.1.65` should be updated to the Milestone `0.1.66`.
+<!-- -->
+- All Issues which are open at the moment the stabilization branch was created won't be included
+in the respective release. Therefore, like Pull Requests, if they have a Milestone defined, the
+Milestone must be updated to refer the next release.
+<!-- -->
+- Close the **Milestone** related to the current stabilization phase.
+    - For example, if the Milestone `0.1.66` was just created, the Milestone `0.1.65` should be closed.
+    - This makes the Milestone less visible and reduces the chance of PRs and Issues being
+    accidentally added to it.
+    _NOTE: It is still possible to add items to the closed Milestone. Just select it in the `closed` tab._
+<!-- -->
+- Open a PR to bump the version of `CMakeLists.txt` on the **master** branch.
+    - For example, when the `stabilization-v0.1.65` is created, the line `set(SSG_PATCH_VERSION 65)`
+    must be updated to `set(SSG_PATCH_VERSION 66)` in the `CMakeLists.txt` file
+    - Reference: https://github.com/ComplianceAsCode/content/pull/9600
+<!-- -->
+- Announce the start of stabilization period on the following communication channels:
+    - Gitter
+        - https://gitter.im/Compliance-As-Code-The/content
+    - SCAP Security Guide Mail List
+        - scap-security-guide@lists.fedorahosted.org
+    - Github Discussion
+        - https://github.com/ComplianceAsCode/content/discussions/new?category=announcements
+    - Here is an example message that could be used as reference:
+    ```
+    Subject: stabilization of v0.1.65
+
+    Hello all,
+
+    The release of Content version 0.1.65 is scheduled for December 2nd.
+    As part of the release process, a stabilization branch was created.
+    Issues and PRs that were not solved were moved to the 0.1.66 milestone.
+
+    Any bug fixes you would like to include in release 0.1.65 should be proposed
+    for the *stabilization-v0.1.65* and *master* branches as a measure to avoid
+    potential conflicts between these branches.
+
+    The next version, 0.1.66, is scheduled to be released on February 3rd,
+    with the stabilization phase starting on January 23th, 2022.
+
+    Regards,
+    ```
+
+## During the stabilization
+
+### Bug Fixes
+- Run whatever extra tests you have and report bugs as Upstream issues using the
+[General Issue](https://github.com/ComplianceAsCode/content/issues/new?assignees=&labels=&template=general_issue.md) template.
+    - Propose bug fixes by targeting the PRs to the **stabilization** branch. e.g. `stabilization-v0.1.65`
+    - Once the PR is **merged** in the **stabilization** branch, cherry-pick it to the **master** branch.
+    For example:
+    ```
+    git checkout master
+    git pull upstream master
+    git checkout -b backport_from_stabilization
+    git cherry-pick abcd1234
```

Comment:
This is not clear about what commit should be cherry picked.
Moreover, I don't think cherry picking is the way to go.
If the PR had one commit we can cherry pick it. But what if it had multiple commits? I don't like much the idea of cherry-picking Merge commits to create a PR.

What I usually do is:
1. Try to create both PRs from a single branch. This is only possible when the files  in both branches have not diverged changed, or any changes in them won't cause conflicts.
```
stab_base=$(git merge-base  master stabilization-v0.1.64)
git checkout -b my-fix $stab_base
# Do the fix, commit and create both PRs through GitHub Web interface
```
2. Fix on stabilization, then rebase all the commits to a new branch from master.
```
# This assumes that stabilization-v0.1.64 is up to date,
# and branch my-fix was branched from stabilization-0.1.64 and has the fixing commits
git checkout my-fix
base_my_fix=$(git merge-base my-fix stabilization-v0.1.64)
git rebase --onto master $base_my_fix HEAD
git checkout -b my-fix-on-master
# Push my-fix and my-fix-on-master and create PRs
```

Reply:
There is a note below highlighting that there are different ways to do it and the cherry-picking is just "one" example. The decision on how to do it is on the author's call. That said, I am fine to include one more example for PRs with multiple commits. This second example could be useful and these two examples should be more than enough.

Reply:
If you want to include only one example, I'd prefer that the first one be included. As it can simplify a lot the creation of PRs for master and stabilization with a single branch.

But why not include all three of them?

Reply:
I wrote that "I am fine to include one more example", not to keep "only one example". : )
The technical details on how do do it is on the author's freedom to decide. Since the fix is sent to both branches, it is all fine.
The examples are intended to clarify the goal and not to confuse it. So, IMO, two simple examples are pretty much enough here. Could you send a commit proposal to amend this section with your example, please? Or send it to me and I include it in the next update.

Reply:
I can add the two examples. Should I wait for any update you may be doing in this branch?

Reply:
It is updated from my side.

Reply:
@marcusburghardt I've added a few commits, please take a look.